main/options/options.cc

```diff
@@ -1109,6 +1117,11 @@ void readOptions(Options &opts,
         if (sorbet::debug_mode) {
             opts.suggestSig = raw["suggest-sig"].as<bool>();
         }
+        if (sorbet::track_untyped_blame_mode) {
+            opts.printBlameUntyped = raw["blame-untyped"].as<bool>();
+            opts.untypedBlameFilePath = raw["blame-untyped-path"].as<string>();
+            opts.trackUntyped = opts.trackUntyped || opts.printBlameUntyped;
```

Comment:
I'm generally not a fan of one option implying another option, because it can cause confusion. You'll notice that we have lots of code like this:

https://github.com/sorbet/sorbet/blob/a7cdaa9c196ff1c8284a19ce7781a9d9c2ff37ea/main/options/options.cc#L837-L842

which avoids assuming implicit options in favor of reporting an error during arg parsing saying to explicitly pass the other arg.